clowdapp.yaml

```diff
@@ -0,0 +1,167 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: drift-backend
+
+objects:
```

Comment:
need to add a dependencies with the list of services your application depends on. See an example here: 

https://github.com/RedHatInsights/compliance-backend/pull/866/commits/085c3fcafdfd1509d0e8ef74e3c3208a594edf08#diff-0f518b5985b4ea86412d39c9757171411345db257cb061b84bb314a7a2b3d5e9R27

and for reference: https://redhatinsights.github.io/clowder/clowder/dev/api_reference.html

search for dependencies, there’s hard and optional dependencies 

the hard ones, means the deployment will wait for those services to be available 
the optional will not make your deployment to wait. however, once the optional dependencies are ready, your pods will be recreated to reload the configuration of the new service.

Reply:
Thanks for reviewing, Victor.  I've made some updates.  I'm sure it'll need further fixes/refining as we move forward.